infrastructure-tests/runTests.sh

```diff
@@ -0,0 +1,53 @@
+#!/bin/bash -e
+
+# Sanity check
+if [ -z "$CX_INFRA_IT_CF_USERNAME" ]; then
+    echo "Failure: Variable CX_INFRA_IT_CF_USERNAME is unset"
+    exit 1
+fi
+
+if [ -z "$CX_INFRA_IT_CF_PASSWORD" ]; then
+    echo "Failure: Variable CX_INFRA_IT_CF_PASSWORD is unset"
+    exit 1
+fi
+
+set -x
+
+# Start a local registry, to which we push the images built in this test, and from which they will be consumed in the test
+docker run -d -p 5000:5000 --restart always --name registry registry:2 || true
+find ../cx-server-companion -type f -exec sed -i "" -e 's/ppiper/localhost:5000\/ppiper/g' {} \;
```

Comment:
why do we need this? what is the desired behavior?

Reply:
We need this to use the images we've built locally so they are consumed from our local registry, not from docker hub.

Reply:
`find ../cx-server-companion -type f -exec sed -i -e 's/ppiper/localhost:5000\/ppiper/g' {} \;` ? empty string after `-i` tries to read the file and throws error `sed: can't read : No such file or directory`

Reply:
Seems to work in travis, and that is all I can check